ambry-cloud/src/main/java/com/github/ambry/cloud/azure/AzureCloudConfig.java

```diff
@@ -32,6 +43,7 @@
   public static final String COSMOS_KEY_SECRET_NAME = "cosmos.key.secret.name";
   public static final String COSMOS_VAULT_URL = "cosmos.vault.url";
   public static final String COSMOS_DIRECT_HTTPS = "cosmos.direct.https";
+  public static final String AZURE_STORAGE_ACCOUNT_INFO = "azure.storage.account_info";
```

Comment:
Change this to `azure.storage.account.info`

Reply:
Done